### ISSUE TITLE
[Accessibility] - Narrator + New Edge. Link dialog is not announced when opened

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor4#5166]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open CKEditor sample in New Edge
2. Open Narrator
3. Focus editor and press Alt+F10
4. Tab through keys until you reach Link button
5. Press Enter to open the dialog

### Expected result

Link dialog name is announced just like in case of e.g. Table Properties dialog.

### Actual result

Link dialog is not announced. Either nothing is announced or button information is announced once more.

## Other details

* Browser: New Edge
* OS: Windows
* CKEditor version: 4.18.0
* Installed CKEditor plugins: Link

**Comment by Comandeer** [comment]
I can confirm the issue, however in my Narrator it's present in nearly every dialog :thinking: 

The issue seems to be connected with the Narrator's focus – it does not jump into the dialog upon opening it but it stays on the button in the toolbar. Not sure why, though. It needs more investigation.

**Comment by jacekbogdanski** [comment]
It is likely that these issues are caused by the fault of the Narrator software, not CKEditor 4. We are officially using the latest Firefox version with the latest JAWS as our reference environment and officially supported platform for accessibility software ([see documentation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#supported-platforms)). Therefore, these issues are not on our priority list at the moment as they require a significant commitment to the research.

**Comment by deeptijoshimsft** [comment]
@Comandeer - Thank you for acknowledging this bug. Can we please have an ETA on when this fix could be available?

**Comment by jacekbogdanski** [comment]
@deeptijoshimsft at the moment we can't provide any ETA for this issue.

Looking at our initial investigation, it's likely that the issue is caused by the fault of the Narrator software, not CKEditor 4. From what I know the issue is not reproducible on other screen readers, and we are officially using the latest Firefox version with the latest JAWS as our reference environment and officially supported platform for accessibility software ([see documentation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#supported-platforms)).

This issue will require some good investigation from our side but due to other priorities, this ticket has to be postponed. We are also rather focused on producing valid HTML from the accessibility point of view than implementing workarounds 
 for 3rd party software limitations, but at the moment it's hard to say if the issue is indeed caused by Narrator or CKEditor 4.
